test/components/report/iframe-answer_spec.js

```diff
@@ -3,7 +3,18 @@ import { shallow } from "enzyme";
 import { Map } from "immutable";
 import { IframeAnswer } from "../../../js/components/report/iframe-answer";
 
-const getReportItemAnswerMock = jest.fn();
+const getReportItemAnswerMock = jest.fn(() => {
+    return [
+      {
+        type: "attachment",
+        name: "test_attachment"
+      },
+      {
+        type: "answerText"
+      }
+    ];
+  }
+);
```

Comment:
`getReportItemAnswerMock` is used to send a postMessage request, it doesn't really return anything. So, this value here will be ignored by the IframeAnswer component. It could be an empty function too. I guess it'd be better not to cause any confusion.

IframeAnswer expects results to be available in the `reportItemAnswer` prop.